### ISSUE TITLE
Env variables added to app deployment

### ISSUE THREAD

**Issue by SarahJaine** [HHS/Head-Start-TTADP#47]
TLDR: Pass env variables to application in deployment pipeline

Background: In https://github.com/HHS/Head-Start-TTADP/issues/12 the continuous deployment (CD) pipeline was added, and the application was [successfully deployed](https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/230/workflows/6b5cf689-4975-4e07-b9cf-054a9e4ff23b/jobs/669) to the dev environment. However, the application does not function because required application variables are missing.

Evidence application is working:
- dev url serves application https://tta-smarthub-dev.app.cloud.gov/
- cloud.gov instance is healthy
